mne_bids/utils.py

```diff
@@ -68,7 +70,18 @@ def get_entity_vals(bids_root, entity_key):
         Path to the root of the BIDS directory.
     entity_key : str
         The name of the entity key to search for. Can be one of
-        ['sub', 'ses', 'run', 'acq'].
+        ['sub', 'ses', 'task', 'run', 'acq'].
+    ignore_sub : iterable | None
```

Comment:
why not allow `str` as well?

Reply:
Because I'm lazy and wanted to avoid having to do a type check ;) But yes, I guess that could make sense! Will add something.

Reply:
I know :) I pointed it out because I've seen many beginners stumble with this.